vyper/compiler/phases.py

```diff
@@ -145,9 +146,33 @@ def vyper_module(self):
         _, ast = self._generate_ast
         return ast
 
+    @cached_property
+    def _resolve_imports(self):
+        vyper_module = copy.deepcopy(self.vyper_module)
```

Comment:
can we add a comment that we're annotating with `ImportInfo` within `resole_imports` and thus the `deepcopy`

Reply:
f94eebc61771e7244ffa682744ad8fdb67ef9a06